README.md

```diff
@@ -524,6 +524,15 @@ The following are optional:
 * `show_hints_on_error`: On progress check failure, display the tests' standard
   error streams as hints. Default is `True`.
 
+* `show_feedback`: On progress check, show feedback on how many tasks out of total 
+  are completed. When `show_hints_on_error` is set to `True`, feedback will always
+  be displayed and setting it to `False` here will not take effect.
+  Default is `True`.
```

Comment:
Intuitively I think of `show_feedback` as being higher-order than `show_hints_on_error`, so my thought would be to flip the logic (and the order) here and instead make it such:

```markdown
* `show_feedback`: On progress check, show feedback on how many tasks out of total 
  are completed. 

* `show_hints_on_error`: On progress check failure, display the tests' standard
  error streams as hints.
  When `show_feedback` is set to `False`, hints will never be displayed and setting
  this to `True` will have no effect.
```

Also I think you don't need to put "Default is" into the prose as Studio will show the default anyway.

Reply:
I think that is a good point, I will implement the change.